gramps_webapi/app.py

```diff
@@ -122,31 +115,39 @@ def create_app(config: Optional[Dict[str, Any]] = None, config_from_env: bool =
             ignore_lock=app.config["IGNORE_DB_LOCK"],
         )
 
+
     if app.config["TREE"] == TREE_MULTI and not app.config["MEDIA_PREFIX_TREE"]:
         warnings.warn(
             "You have enabled multi-tree support, but `MEDIA_PREFIX_TREE` is "
             "set to `False`. This is strongly discouraged as it exposes media "
             "files to users belonging to different trees!"
         )
 
+
     if app.config["TREE"] == TREE_MULTI and app.config["NEW_DB_BACKEND"] != "sqlite":
-        # needed in case a new postgres tree is to be created
+                # needed in case a new postgres tree is to be created
         gramps_config.set("database.host", app.config["POSTGRES_HOST"])
         gramps_config.set("database.port", str(app.config["POSTGRES_PORT"]))
 
     # load JWT default settings
     app.config.from_object(DefaultConfigJWT)
-
-    # instantiate JWT manager
     JWTManager(app)
 
+
     app.config["SQLALCHEMY_DATABASE_URI"] = app.config["USER_DB_URI"]
     user_db.init_app(app)
 
+
     request_cache.init_app(app, config=app.config["REQUEST_CACHE_CONFIG"])
+
+
+    configure_oauth(app)
```

Comment:
This will register the providers.

Does this involve an HTTP request to the provider endpoint? If so, what if that request fails? Will the whole flask app fail to start?